src/app/components/core/logs/notification-log/notification-log.component.ts

```diff
@@ -58,6 +58,12 @@ export class NotificationLogComponent implements OnInit, OnDestroy {
     this.subscription = interval(this.refreshInterval)
       .pipe(takeWhile(() => this.isAlive), takeUntil(this.destroy$)) // only fires when component is alive
       .subscribe(() => {
+        if (this.sourceName) {
+          const quickView = <HTMLDivElement>document.getElementById('quickviewDefault');
+          if (!quickView.classList.contains('is-active')) {
+            return;
+          }
+        }
```

Comment:
By this check we are only returning from the Interval Observable but the Interval() Observable will stay active in the background. To stop the Interval Observable completely on page close we need to see how we can stop that Observable. To do that we need to unsubscribe the this.subscription.